### DISCUSSION TITLE
Collection Product filtering by Price not working when using localisation

### DISCUSSION THREAD

**Discussion by ghost** [Shopify/storefront-api-feedback#188] [Help]
For the given query when using the default locale (that store uses) which is in my case `HR` both for country and language the product filtering by price works as expected. But when using a country like 'US' it ignores the price filter but other filters work (in my case product type, vendor, variant size). I also checked for countries like Serbia, Macedonia and it still didn't work as expected. The only time it works for the US (or any other country I'm presuming) is when using a single currency instead of a given country's currency

```
query ProductsCollectionByHandle(
    $handle: String!
    $numberOfProducts: Int!
    $cursor: String
    $country: CountryCode
    $language: LanguageCode
    $productFilters: [ProductFilter!]
  ) @inContext(country: $country, language: $language) {
  collection(handle: $handle) {
      id
      handle
      title
      products(
        first: $numberOfProducts
        filters: $productFilters
        after: $cursor
      ) {
      .
      .
      .
```

Anyone experienced it or have any fix for it? Even paid plugins?

**Comment by Huzaifa-Sajjad** [comment]
Hey, did you fix this issue by any chance? I am facing the same issue.

**Reply by JoaoPauloXD** [reply]
Hello, did you find a solution? I am on this issue too.

**Reply by Huzaifa-Sajjad** [reply]
Hey, yeah I found the solution. It is mentioned in the Hydrogen documentation https://shopify.dev/docs/storefronts/headless/hydrogen/markets 

You need to follow the following steps:

1. Setup a default locale (It should be the default currency of the Shopify store) setting this will fix the issue.
2. Setup a multi-region and multilingual storefront
